client/verta/verta/runtime.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict
+import json
+import threading
+import os
+
+
+_THREAD = threading.local()
+
+def init_thread_context() -> None:
+    """Initialize our thread-local variable for storing logging context."""
+    if not hasattr(_THREAD, "context"):
+        _THREAD.context = defaultdict()
```

Comment:
I think `.context` might not be the right name. To me, _this_ isn't "the context"; this is just accumulated logs. But no strong feelings, here.

Reply:
Refactored to use "logs" instead.  Seems more intuitive.